src/styles/components/circle-tag.scss

```diff
@@ -0,0 +1,40 @@
+.circle-tag {
+  height: 20px;
+  width: 20px;
+  border-radius: 50%;
+  margin-right: 0.4rem;
+  margin-left: -1.5rem;
+  margin-bottom: -1.3rem;
+  display: flex;
+  background-color: repairs-hub-colour('blue');
+
+  &--status-in-progress {
+    background-color: repairs-hub-colour('blue');
+  }
+  &--status-work-complete,
+  &--status-locked {
+    background-color: repairs-hub-colour('white');
+    border: 1px solid black;
+  }
+  &--status-work-cancelled {
+    background-color: repairs-hub-colour('grey');
+  }
+  &--status-pending-approval {
+    background-color: repairs-hub-colour('yellow');
+  }
+  &--status-variation-pending-approval {
+    background-color: repairs-hub-colour('yellow');
+  }
+  &--status-authorisation-pending-approval {
+    background-color: repairs-hub-colour('yellow');
+  }
```

Comment:
Very small nitpick but we could reduce the above three rules into one rule with three selectors if we wanted..